config/peerpods/podvm/aws-podvm-image-handler.sh

```diff
@@ -318,6 +439,33 @@ function create_ami() {
 
     # Create AWS ami using packer
     create_ami_using_packer
+}
+
+# Function to create the ami in AWS
+
+function create_ami() {
+    echo "Creating AWS AMI"
+
+    # Create the AWS image
+    # If any error occurs, exit the script with an error message
+
+    # Install packages if INSTALL_PACKAGES is set to yes
+    if [[ "${INSTALL_PACKAGES}" == "yes" ]]; then
```

Comment:
since you are adding this installation steps here (which makes sense) I believe we have it duplicated under creater_ami_from_scratch() method. IIUC, when following the from scratch flow, it would be installing twice. Should we remove there? Or am I missing something?